src/ports/catalog/component.ts

```diff
@@ -26,26 +23,12 @@ export function createCatalogComponent(options: {
   const { database, favoritesComponent } = options
 
   async function fetch(filters: CatalogOptions) {
-    const { network, creator } = filters
-    const marketplaceChainId = getMarketplaceChainId()
-    const collectionsChainId = getCollectionsChainId()
+    const { network } = filters
     let catalogItems: Item[] = []
     let total = 0
     const client = await database.getPool().connect()
     try {
-      const sources = (
-        network
-          ? [network]
-          : creator
```

Comment:
the issue was in this line, `creator` won't ever be falsy since it's an array (either empty or with a list of adddresses)